src/ui/flutter_app/lib/main.dart

```diff
@@ -46,6 +47,8 @@ Future<void> main() async {
   debugPrint('Environment [dev_mode]: ${EnvironmentConfig.devMode}');
   debugPrint('Environment [monkey_test]: ${EnvironmentConfig.monkeyTest}');
 
+  enableFlutterDriverExtension();
```

Comment:
This should only be done when a testing apk is build.
See the documentation 
```
[...]
Applications intended for release should never include this method.
[...]
```

Also catcher should also be disabled as it will catch the errors the test should find :upside_down_face:

Reply:
OK! I use `${EnvironmentConfig.test}` (Renamed from `${EnvironmentConfig.monkeyTest})` to control this.